features/machine/machine.feature

```diff
@@ -303,4 +303,28 @@ Feature: Machine features testing
       | ["spec"]["template"]["metadata"]["labels"]["machine.openshift.io/cluster-api-machineset"] | default-valued-33058                            |
     Then the step should succeed
 
+  # @author miyadav@redhat.com
+  # @case_id OCP-33455
+  @admin
+  Scenario: Run machine api Controllers using leader election
+    Given I have an IPI deployment
+    And I switch to cluster admin pseudo user
+    And I use the "openshift-machine-api" project
+
+    Given a pod becomes ready with labels:
+      | api=clusterapi, k8s-app=controller |
+
+    And I saved following keys to list in :containers clipboard:
+      | machine-controller      | |
+      | machineset-controller   | |
+      | nodelink-controller     | |
+
+   Then I repeat the following steps for each :id in cb.containers:
+   """
+   When I run the :logs admin command with:
+      | resource_name | <%= pod.name %> |
+      | c             | #{cb.id}        |
+    Then the output should contain:
+      | attempting to acquire leader lease  openshift-machine-api/cluster-api-provider |
```

Comment:
@miyadav should there be two spaces betweeen ```leader lease  openshift-machine-api/cluster-api-provider```

Reply:
Nice, catch.

Reply:
yeah it is like that  :) ..

Reply:
@miyadav ok, that's probably a bug...nitpicking kind...very low priority